src/App.tsx

```diff
@@ -70,11 +71,15 @@ interface AppProps {
 export const App: FC<AppProps> = ({ history }) => {
   const [loaded, setLoaded] = useState(false);
   useEffect(() => {
-    Initializer.init()?.then(() => {
-      if (loaded) return;
-      setLoaded(true);
-      widget?.api.sendContentLoaded();
-    });
+    Initializer.init()
+      ?.then(async () => {
+        if (loaded) return;
+        setLoaded(true);
+        await widget?.api.sendContentLoaded();
+      })
+      .catch((e) => {
+        logger.error(e);
+      });
```

Comment:
If there are enough loggers that simply log the caught error variable without modifying it or adding some other text to the logged message, they could be made into one-liners:
```suggestion
      .catch(logger.error);
```